#RUN THIS CODE TO CHUNK UP THE FASTQTL COMMANDS - THIS IS TO PARALLELIZE THE COMMAND
fastQTL --vcf samples.vcf.gz '#THIS WILL BE THE GENOTYPES OF YOUR SAMPLES IN VCF FORMAT' \ 
--bed expression.bed.gz '#THIS WILL BE THE PHENOTYPES, IN OUR CASE GENE EXPRESSION, IN BED FORMAT' \
--out PATH/TO/OUTPUT '#THIS WILL BE THE OUTPUT DIRECTORY'\
--commands 500 commands.500.txt '#THIS WILL BE A LIST OF THE COMMANDS, CHANGE THE NUMBER BASED ON HOW MANY CHUNKS YOU WANT TO PARALLELIZE' \
--window 1e6 '#THIS WILL BE THE WINDOW YOU SET FOR THE ANALYSIS, THIS IS THE DEFAULT AND WHAT WE USED'

#THIS WILL CREATE A TEXT FILE WITH 500 COMMANDS, REPRESENTING 500 REGIONS THAT FASTQTL WILL RUN ON
#THESE 500 COMMANDS WILL HAVE THE OPTIONS LISTED ABOVE
#YOU CAN THEN RUN THOSE COMMANDS IN PARALLEL
